### PR TITLE
Added Litre/min and milliLitre/min in Flow

### PR DESCRIPTION
**Pull request by flopp999**

### CODE REVIEW

File: public/app/core/utils/kbn.ts [hunk 1]
```diff
@@ -1075,6 +1077,8 @@ kbn.getUnitFormats = function() {
         { text: 'Cubic feet/sec (cfs)', value: 'flowcfs' },
         { text: 'Cubic feet/min (cfm)', value: 'flowcfm' },
         { text: 'Litre/hour', value: 'litreh' },
+        { text: 'milliLitre/min (mL/min)', value: 'flowmlpm' },
+        { text: 'Litre/min (l/min)', value: 'flowlpm' },
```
Comment (marefr) [inline public/app/core/utils/kbn.ts]: Litre/min should be after Litre/hour

File: public/app/core/utils/kbn.ts [hunk 2]
```diff
@@ -575,13 +575,15 @@ kbn.valueFormats.m3 = kbn.formatBuilders.fixedUnit('m3');
 kbn.valueFormats.Nm3 = kbn.formatBuilders.fixedUnit('Nm3');
 kbn.valueFormats.dm3 = kbn.formatBuilders.fixedUnit('dm3');
 kbn.valueFormats.gallons = kbn.formatBuilders.fixedUnit('gal');
-
+
```
Comment (marefr) [inline public/app/core/utils/kbn.ts]: You have a trailing whitespace here that made the frontend test fail. Please remove

**Review by marefr — CHANGES_REQUESTED** [review]

**Comment by flopp999** [comment]
Changed made and tests are OK

**Review by marefr — APPROVED** [review]

**Comment by marefr** [comment]
Thank you for contributing to Grafana!

**Pull request merged**